pkg/service/server.go

```diff
@@ -220,8 +220,12 @@ func (s *LivekitServer) Start() error {
 	if s.config.RTC.TCPPort != 0 {
 		values = append(values, "rtc.portTCP", s.config.RTC.TCPPort)
 	}
-	if !s.config.RTC.ForceTCP && s.config.RTC.UDPPort != 0 {
-		values = append(values, "rtc.portUDP", s.config.RTC.UDPPort)
+	if !s.config.RTC.ForceTCP && (s.config.RTC.UDPPort != 0 || s.config.RTC.UDPPorts.Valid()) {
+		if s.config.RTC.UDPPort != 0 {
+			values = append(values, "rtc.portUDP", s.config.RTC.UDPPort)
```

Comment:
Oh, I made a comment about `udp_ports` overriding `udp_port`. But, looks like it is the other way around here. Would be good to mention that in config which takes precedence.

Reply:
UDP ports will takes precedence, will change the log